Tests/FirebaseTests/Tests/DynamicLinks/DynamicLinksEndpointsTests.swift

```diff
@@ -0,0 +1,21 @@
+import XCTest
+@testable import Firebase
+@testable import Vapor
```

Comment:
do you really need access to the internal of Vapor here?

Reply:
58c04a9